Bang.Tests/DomainUnitTests/GameTests.cs

```diff
@@ -9,19 +10,79 @@ namespace Bang.Tests.DomainUnitTests
 {
     public class GameTests
     {
-        
+        [Fact]
+        public void Drop_cards_removes_cards_from_hand()
+        {
+            var game = CreateAndStartGame();
+            var player = game.Players.First();
+            var cardsToDrop = player.PlayerHand.Take(2).ToList();
+            game.Gameplay.DropCardsFromHand(cardsToDrop, player.Id);
+
+            foreach(var card in cardsToDrop)
```

Comment:
You know that there are two cards to drop. 
So it will be better replace cycle into second assert statement
Assert.DoesNotContain(card[0], player.PlayerHand);
Assert.DoesNotContain(card[1], player.PlayerHand);

There is another option. You can drop only one card, not two. It will make test more readable.